src/components/routers/WebhookRouter.tsx

```diff
@@ -273,7 +273,7 @@ export default class WebhookForm extends React.Component<WebhookRouterFormProps,
         let hasEmpty = false;
 
         for (const header of newHeaders) {
-            if (header.name.trim().length === 0 && header.value.trim().length === 0) {
+            if (!header.name.trim().length && !header.value.trim().length) {
```

Comment:
Not saying don't do this. Just sharing my bias is generally to avoid truthy/falsy and defer to more explicit conditions. While it is equal here, experience as a practice is this can lead to unintended consequences when it really isn't buying you anything. ie, you may not want munge the handling of false, 0, NaN, and null, but might not consider them consistently with this practice.

Reply:
Also, I believe HTTP headers can have empty values, so perhaps we should allow named headers without a value set.

Reply:
gotcha, will revert. I believe we do allow for headers to contain a name but no value per `HeaderElement`'s `validate` method.

Reply:
http://gph.is/2Eq5ph3